app/networkapi/features/apps.py

```diff
@@ -2,4 +2,8 @@
 
 
 class FeaturesConfig(AppConfig):
-    name = 'features'
+    name = 'networkapi.features'
+    verbose_name = 'features'
+
+    def ready(self):
+        from networkapi.features import signals  # noqa
```

Comment:
This asks flake8 to ignore the unused import on this line. 

Pro: less code
con: relies on the side-effects of the import, which sets up the signals.

We could make the signals module export a function and call it, if we wanted to avoid depending on side-effects.